glib/src/win32.rs

```diff
@@ -4,18 +4,22 @@ use std::path::PathBuf;
 
 use crate::{translate::*, GString, StrV};
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
-pub enum OSType {
```

Comment:
maybe keep this enum as an alias?

Reply:
That seems like a bad idea. It's specifically a Windows OS type, why should we drop the prefix of it? :)

Reply:
I think we will later have it as `glib_win32::OSType` or so but that's for some other time.

Reply:
We can leave it as is then